### ISSUE TITLE
MacOS Sierra Install problems

### ISSUE THREAD

**Issue by steel99xl** [k4m4/kickthemout#57]
After following the installation instruction I keep going this problem and i was wondering if there is any way to fix it
Thanks
![screen shot 2017-01-30 at 09 34 33](https://cloud.githubusercontent.com/assets/25434505/22427054/9db47de6-e6cf-11e6-8c8a-4e3ef8b5541a.png)

**Comment by Mrdev2017** [comment]
I get the same error every time, got no idea how to fix it so I am stuck with you here my friend :)

**Comment by steel99xl** [comment]
i found out how to fix it 
export PYTHONPATH=`brew --prefix`/lib/python2.7/site-packages:$PYTHONPATH
this redirects the python directory and fixes the problem

**Comment by Mrdev2017** [comment]
Thanks man really appreciate it ;)

**Comment by Mrdev2017** [comment]
sorry for asking more questions but i am a beginner developer, i can't run that command or do i have to do  it in separate parts?

**Comment by steel99xl** [comment]
no you should not 
what is the error it is giving you

**Comment by Mrdev2017** [comment]
![schermafbeelding 2017-01-30 om 18 56 34](https://cloud.githubusercontent.com/assets/25437343/22434798/db2edd76-e71d-11e6-9ae5-97dbf7293ec7.png)

**Comment by Mrdev2017** [comment]
i installed requirements

**Comment by steel99xl** [comment]
did you run the
 command 
"export PYTHONPATH=brew --prefix/lib/python2.7/site-packages:$PYTHONPATH"

**Comment by Mrdev2017** [comment]
yes and it gave me this:

**Comment by steel99xl** [comment]
i dont see any thing but you might have to relaunch your terminal

**Comment by Mrdev2017** [comment]
![schermafbeelding 2017-01-30 om 18 58 10](https://cloud.githubusercontent.com/assets/25437343/22434864/1cf58494-e71e-11e6-9ae4-582b5c8ebc03.png)

**Comment by Mrdev2017** [comment]
oh  that might be the case

**Comment by steel99xl** [comment]
what version of python do you have
if you don't know do   python -V

**Comment by Mrdev2017** [comment]
2.7.13

**Comment by k4m4** [comment]
Hi there. You are following the wrong installation instructions. Head over [here](https://github.com/k4m4/kickthemout/#mac-os-x-installation) to install KickThemOut on a mac. Thanks.

**Comment by Mrdev2017** [comment]
hi K4M4 thank you  for helping. I will follow the instructions you send.

**Comment by k4m4** [comment]
No problem! Anytime. Let me know if you have any further issues.

**Comment by FrancescoSaverioZuppichini** [comment]
Hi, 

```bash
sudo pip install pcapy
```

give me error:

```
Command "/usr/bin/python -u -c "import setuptools, tokenize;__file__='/private/tmp/pip-build-eT83Ra/pcapy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-1VYCNq-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /private/tmp/pip-build-eT83Ra/pcapy/
```
 as well as brew one. 

```
Error: No available formula with the name "scapy"
```

I've re-installed Sierra so there is no way that the error is caused by other thing.

**Comment by steel99xl** [comment]
Form what I tried you have to install it using    brew

**Comment by nicosandller** [comment]
was @FrancescoSaverioZuppichini issue solved? I get the same error and can't find a solution. pcapy won't install.

**Comment by ghattab** [comment]
How can you close this if it isn't solved...?? I can reproduce the issue Francesco and Nico are having under MacOS High Sierra.

**Comment by k4m4** [comment]
Hi there @nicosandller & @ghattab. This issue was closed as it is appears to be a duplicate of #10. Thanks!
